src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,32 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+
+import usersFromServer from '../../api/users';
+
+type Props = {
+  todo: Omit<Todo, 'user'>;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: { title, completed, userId },
+}) => {
+  const user = userId
+    ? usersFromServer.find(usr => usr.id === userId)
+    : null;
```

Comment:
no needs to make a condition since you get falsy value if you don't find a user anyway
```suggestion
  const user =  usersFromServer.find(usr => usr.id === userId)
```